core/ecschema-editing/src/Editing/Editor.ts

```diff
@@ -6,37 +6,76 @@
  * @module Editing
  */
 
-import { assert } from "@bentley/bentleyjs-core";
-import { SchemaContext } from "../Context";
-import { DelayedPromiseWithProps } from "../DelayedPromise";
-import {
-  ConstantProps, CustomAttributeClassProps, EntityClassProps, EnumerationPropertyProps, EnumerationProps, FormatProps, InvertedUnitProps,
-  KindOfQuantityProps, MixinProps, PhenomenonProps, PrimitiveArrayPropertyProps, PrimitivePropertyProps, PropertyCategoryProps,
-  RelationshipClassProps, StructArrayPropertyProps, StructClassProps, StructPropertyProps, UnitProps, UnitSystemProps,
-} from "../Deserialization/JsonProps";
-import { CustomAttributeContainerType, ECClassModifier, PrimitiveType, SchemaItemType, SchemaMatchType, StrengthDirection } from "../ECObjects";
-import { ECObjectsError, ECObjectsStatus } from "../Exception";
-import { ECClass, MutableClass, MutableStructClass, StructClass } from "../Metadata/Class";
-import { MutableConstant } from "../Metadata/Constant";
-import { CustomAttribute } from "../Metadata/CustomAttribute";
-import { CustomAttributeClass, MutableCAClass } from "../Metadata/CustomAttributeClass";
-import { EntityClass, MutableEntityClass } from "../Metadata/EntityClass";
-import { AnyEnumerator, Enumeration, MutableEnumeration } from "../Metadata/Enumeration";
-import { Format, MutableFormat } from "../Metadata/Format";
-import { InvertedUnit, MutableInvertedUnit } from "../Metadata/InvertedUnit";
-import { MutableKindOfQuantity } from "../Metadata/KindOfQuantity";
-import { Mixin, MutableMixin } from "../Metadata/Mixin";
-import { OverrideFormat } from "../Metadata/OverrideFormat";
-import { MutablePhenomenon, Phenomenon } from "../Metadata/Phenomenon";
-import { MutablePropertyCategory } from "../Metadata/PropertyCategory";
-import { MutableRelationshipClass, RelationshipClass } from "../Metadata/RelationshipClass";
-import { MutableSchema, Schema } from "../Metadata/Schema";
-import { MutableUnit, Unit } from "../Metadata/Unit";
-import { MutableUnitSystem, UnitSystem } from "../Metadata/UnitSystem";
-import { SchemaItemKey, SchemaKey } from "../SchemaKey";
-import { FormatType } from "../utils/FormatEnums";
 import * as Rules from "../Validation/ECRules";
 
+import {
+  AnyEnumerator,
```

Comment:
Don't do one import per line, if it's too long for one line just make it a couple of lines

Reply:
Ok.  I briefly had this import order extension enabled which auto-ordered a couple of files. I will correct.